Source/OxyPlot/PlotModel/PlotModel.Rendering.cs

```diff
@@ -179,7 +179,10 @@ private static double MaxSizeOfPositionTier(IRenderContext rc, IEnumerable<Axis>
             double maxSizeOfPositionTier = 0;
             foreach (var axis in axesOfPositionTier)
             {
-                var size = axis.Measure(rc);
+                // update axis with latest render context, adjusting dimensions, etc.
+                axis.Update(rc);
```

Comment:
What would you suggest?  I just need a property on Axes which reflects the "desired" or "natural" size (as opposed to the rendered size which may reflect user size overrides).